server/jetstream_test.go

```diff
@@ -9605,7 +9251,6 @@ func TestJetStreamStoreDirectoryFix(t *testing.T) {
 	defer removeDir(t, sd)
 
 	conf := createConfFile(t, []byte(fmt.Sprintf("listen: 127.0.0.1:-1\njetstream: {store_dir: %q}\n", sd)))
-	defer removeFile(t, conf)
```

Comment:
maybe we should make a note that for this test not using t.TempDir unlike other tests and have to rely on os.TempDir